.github/workflows/eks-production-release.yaml

```diff
@@ -0,0 +1,89 @@
+name: Build Plugin Zip and Push to S3
+
+on:
+  release:
+    types:
+      - published
+
+jobs:
+  deploy:
+    name: Empire Wordpress Plugin - Build & Publish Package
+    runs-on: ubuntu-latest
+
+    env:
+      AWS_REGION: us-west-2
+      AWS_BUCKET: empire-wordpress-plugin
+      BUILD_NUMBER: ${{ github.event.release.tag_name }}
+
+    steps:
+      # This is a pre-configured step to send build notifications to slack
+      - uses: voxmedia/github-action-slack-notify-build@v1
+        id: slack
+        with:
+          channel: dev-notifications
+          status: STARTED
+          color: warning
+        env:
+          SLACK_BOT_TOKEN: ${{ secrets.SLACK_BOT_TOKEN }}
+
+      - name: Checkout
+        uses: actions/checkout@v2
+
+      # Install dependencies
+      - name: Composer (php-actions)
+        uses: php-actions/composer@v6
+        with:
+          dev: no
+
+      # Create a packaged up zip file
+      - name: Set Version Number & Build Zip
+        id: build-zip
+        run: |
+          sed -i 's/VERSION/${{ env.BUILD_NUMBER }}/' empire.php
+          mkdir empire
+          ls -alh
+          sudo chown -R runner:docker .
+          mv empire.php empire/
+          mv composer.json empire/
+          mv composer.lock empire/
+          mv vendor empire/
+          zip -r empire-${{ env.BUILD_NUMBER }}.zip empire
+
+      # Set up our S3 Connection
+      - name: Set up S3cmd cli tool
+        uses: s3-actions/s3cmd@v1.1
+        with:
+          provider: aws # default is linode
+          region: ${{ env.AWS_REGION }}
+          access_key: ${{ secrets.AWS_ACCESS_KEY_ID }}
+          secret_key: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
+
+      # Push zip to S3 Bucket
+      - name: Deploy
+        run: |
+          s3cmd put empire-${{ env.BUILD_NUMBER }}.zip s3://${{ env.AWS_BUCKET }}/
+          s3cmd put empire-${{ env.BUILD_NUMBER }}.zip s3://${{ env.AWS_BUCKET }}/empire-latest.zip
```

Comment:
Why are we uploading this to S3? What happens next? Do we manually add it to the release?

Reply:
Yea, I think it would be preferable to upload the package directly to the Release itself and not to S3. I found [this action](https://github.com/marketplace/actions/github-action-publish-binaries) that should be able to do to it as part of the build process.

Reply:
Ah I should add to the release, yes.

I want to put a link to “latest” in the Empire Dashboard and S3 was my initial thought, but Id rather just link to the latest  release.